components/unzipped/hire/hire.js

```diff
@@ -0,0 +1,224 @@
+                    <Label>Project Name</Label>
+                    <Select
+                        value={select}
+                        onChange={(e) => setSelect(e.target.value)}
+                        style={{ width: '470px', border: '1px solid black' }}
+                    >
+                        <MenuItem value="Unzipped">Unzipped</MenuItem>
+                        <MenuItem value="Why">Find Talent</MenuItem>
+                        <MenuItem value="Client">Client</MenuItem>
+                        <MenuItem value="Freelancer">Freelancer</MenuItem>
+                    </Select>
+                    <Label>Send a private message</Label>
+                    <TextareaField rows={50} cols={100} />
+                    <Label>job type</Label>
+                    <InputField type='text' />
+                    <Label>hourly rate (show budget if selected)</Label>
+                    <div style={{ display: 'flex', width: 470, gap: 20 }}>
+                        <div style={{ border: '1px solid black' }}>
+                            <Span>$</Span>
+                            <InputHourlyField value={50} />
+                            <Span>per hours</Span>
+                        </div>
+                        <div>
+                            <Select
+                                value={selectCurrency}
+                                onChange={(e) => setSelectCurrency(e.target.value)}
+                                style={{
+                                    width: '77px',
+                                    border: '1px solid black',
+                                    paddingLeft: 5,
+                                    paddingRight: 5,
+                                    paddingTop: 6,
+                                    fontSize: '12px !important'
+                                }}
+                            >
+                                <MenuItem value="USD">USD</MenuItem>
+                                <MenuItem value="EUR">EUR</MenuItem>
+                                <MenuItem value="GBP">GBP</MenuItem>
+                            </Select>
+                        </div>
+                    </div>
+                    <Label>weekly tracking limit</Label>
+
+                    <div style={{ display: 'flex', width: 353, border: '1px solid black', justifyContent: 'space-between', alignItems: 'center' }}>
+                        <div>
+                            <TrackingField value={50} />
+                        </div>
+                        <div>
+                            <Span>hours / week</Span>
+                        </div>
+
+                    </div>
+                </div>
+                <div style={{ display: 'flex', justifyContent: 'end', paddingRight: '4rem', marginTop: 35 }}>
+                    <HireButton ><ButtonText onClick={() => router.push('/subscribe')} >Hire (user)</ButtonText></HireButton>
```

Comment:
This should push to the /recurring-payment page. It is slightly different than the subscribe page. I referenced that page because there were many components from that page that can be re used.

does creating that page need to be a separate ticket?

Reply:
https://www.figma.com/file/3G9a7FDpxxjp5h0srZnMT0/Unzipped?type=design&node-id=0-1&mode=design&t=Nj0khOl4aWvEqFX0-0

Reply:
Yes, It'd be better to create a new ticket for that page.